app/addons/components/assets/less/layouts.less

```diff
@@ -19,3 +19,11 @@
   flex-direction: row;
   height: 100%;
 }
+
+//yes !important!!! We have to overridde some styles I can't override any other way for now
```

Comment:
That is tricky to do, because its not one specific class styling. It would require a large clean up of our css which is out of scope for this PR.